stake-pool/py/pyproject.toml

```diff
@@ -0,0 +1,67 @@
+[project]
+name = 'stake-pool'
+version = '1.0'
+authors = [
+  { name= 'joncinque'},
+]
+description = 'solana labs stake pool reference implementation'
+readme = 'README.md'
+license = { file='LICENSE' }
+requires-python = '>=3.7'
+classifiers = [
+    'Programming Language :: Python :: 3',
+    'License :: OSI Approved :: Apache Software License',
+    'Operating System :: OS Independent',
+]
+dependencies = [
```

Comment:
It would be better to split these up into dependencies and optional-dependencies, to separate what's needed to use the package vs build / test it. But we can do that another time

Reply:
happy to add that if you can share which dependencies are optional

Reply:
For sure, I just need to do a little sweep of the dependencies, I'll also update a bunch of them while I'm at it